source/docs/software/wpilib-tools/robot-simulation/introduction.rst

```diff
@@ -55,3 +55,18 @@ Your console output in Visual Studio Code should look like the below. However, t
 .. important:: Simulation can also be run outside of VS Code using ``./gradlew simulateJava``. It's important to note that C++ simulation is not available through command-line at this time.
 
 .. note:: macOS users should be aware of the :ref:`simulation fails to launch known issue <docs/2020-overview/known-issues:macOS Simulation fails to launch in Visual Studio Code>`
+
+Running Robot Dashboards
+------------------------
+
+Both Shuffleboard and SmartDashboard can be used with WPILib simulation.
+
+Shuffleboard
+^^^^^^^^^^^^
+
+Shuffleboard is automatically configured to look for a NetworkTable instance from the robotRIO **or from `localhost`**, so once the simulation is loaded, Shuffleboard will connect automatically.
```

Comment:
Are you sure that shuffleboard connects to localhost automatically? It will let you enter localhost as the network tables server, but doesn't connect automatically for me.